pkg/restapi/edv/edverrors/edverrors.go

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
The reason for this "edverrors" file is because I was running into circular dependency issues with my new code, so I needed to move these errors to a new package. Not sure if this is indicative of a larger problem with my code structure or not.